src/tests/ftest/dfuse/simul.py

```diff
@@ -73,7 +73,7 @@ class PosixSimul(DfuseTestBase):
     def run_simul(self, include=None, exclude=None, raise_exception=True):
         """Run simul.
 
-        If an include value is set, the exclude value is ignored and vice versa.
+        If an include value is set, the exclude value is ignored and vice-versa.
```

Comment:
This one is weird. I don't think vice-versa is the correct spelling. It should be "vice versa". Is pylint still complaining after adding "versa" to the dictionary?

Reply:
Included by mistake, I was checking if vice-versa was allowed, I didn't mead to commit this change.  Now reverted.